src/shared/services/auth/services/state-token-generator/state-token-generator.js

```diff
@@ -9,10 +9,10 @@
  * @returns {string} 16 random Ascii characters, empty in case the
  * `crypto` library is not available.
  */
-export default function () {
+const stateTokenGenerator = () => {
```

Comment:
Same here; why is this required? Looks like there is a change in the rules in the linter configuration; I would tackle the issue there instead of changing the code.

Reply:
yes, the default settings of prettier are a bit changed